cli/resource/resolve.go

```diff
@@ -0,0 +1,113 @@
+package resource
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+
+	corev2 "github.com/sensu/sensu-go/api/core/v2"
+	"github.com/sensu/sensu-go/types"
+)
+
+var (
+	// All is all the core resource types and associated sensuctl verbs (non-namespaced resources are intentionally ordered first).
+	All = []types.Resource{
```

Comment:
For reference, this move breaks sensu-enterprise-go: https://github.com/sensu/sensu-enterprise-go/blob/5f171939c1b6a912144b3d664d13c31995fda0ba/cli/commands/commands.go#L31

Reply:
https://github.com/sensu/sensu-enterprise-go/pull/919 resolves this.